mozregression/fetch_configs.py

```diff
@@ -174,47 +173,55 @@ def set_inbound_branch(self, inbound_branch):
         if inbound_branch:
             self.inbound_branch = inbound_branch
 
-    def inbound_base_urls(self):
+    def tk_inbound_route(self, changeset):
+        """
+        Returns a taskcluster route for a specific changeset.
+        """
         raise NotImplementedError
 
 
-class FirefoxInboundConfigMixin(InboundConfigMixin):
-    build_base_os_part = {
-        'linux': {32: 'linux', 64: 'linux64'},
-        'win': {32: 'win32', 64: 'win64'},
-        'mac': {64: 'macosx64'}
-    }
-    root_build_base_url = ('http://inbound-archive.pub.build.mozilla.org/pub'
-                           '/mozilla.org/firefox/tinderbox-builds/%s-%s/')
+def _common_tk_part(inbound_conf):
+    # private method to avoid copy/paste for building taskcluster route part.
+    if inbound_conf.os == 'linux':
+        part = 'linux'
+        if inbound_conf.bits == 64:
+            part += str(inbound_conf.bits)
+    elif inbound_conf.os == 'mac':
+        part = 'macosx64'
+    else:
+        # windows
+        part = '%s%s' % (inbound_conf.os, inbound_conf.bits)
+    return part
 
-    def inbound_base_urls(self):
-        return [self.root_build_base_url
-                % (self.inbound_branch,
-                   self.build_base_os_part[self.os][self.bits])]
+
+class FirefoxInboundConfigMixin(InboundConfigMixin):
+    def tk_inbound_route(self, changeset):
+        return 'buildbot.revisions.%s.%s.%s' % (
```

Comment:
lately I've been more into using format for this sort of thing (i.e. `return 'buildbot.revisions.{}.{}.{}'.format(...)`) as I think it's a little easier to read/understand